src/buffer.rs

```diff
@@ -493,6 +493,7 @@ mod tests {
     /// and the output channels are just 0.
     /// This test assures that when the buffers are zipped together,
     /// the input values do not change.
+    #[allow(clippy::float_cmp)]
```

Comment:
We can add it crate-wide through the CI file.

For right now, I'm hesitant to ignore this particular one crate-wide in case we actually *do* do some strict float compares where they shouldn't be, and having the `allow` statements makes it easy to audit in the future.

Although on the other hand, I can't really think of a scenario where a strict compare would be incorrect and a fuzzy float compare would be correct, when talking about the internals of a VST crate. So... idk?

Reply:
I think the set of enforced lints should only contain ones that are usually a good idea, and where there could be a few exceptions. `float_cmp` is rather the other way around. There are only false positives, and quite a lot of them.

Reply:
Also, I don't find it likely that we would ever need fuzzy float comparisons in this crate, outside of examples. Whereas the exact comparisons are quite proliferate in tests.

Reply:
Yeah, I can get behind this as of right now. Might change my mind if we ever do come across a `float_cmp` that we should have caught :wink: